frontend/apps/filemanager/filemanagermenu.lua

```diff
@@ -260,6 +260,10 @@ function FileManagerMenu:setUpdateItemTable()
             end,
             callback = function()
                 G_reader_settings:flipNilOrFalse("dev_startup_no_fbdepth")
+                local InfoMessage = require("ui/widget/infomessage")
+                UIManager:show(InfoMessage:new{
+                    text = _("This will take effect on next restart."),
```

Comment:
Might as well write something more specific, like `The new bit depth will take effect after restarting KOReader.` or `KOReader will need to restart to enable the new bit depth.`

Reply:
Incidentally, I've been thinking that messages of this type could also consider using a ConfirmBox or something instead, with the cancel/OK buttons reading `Restart later` and `Restart now`.

Reply:
I mainly went with that generic wording because it's already used elsewhere, so, already translated ;).

Reply:
That's true. On the flipside, the previous translation will show up as an  `xx %` match so it can be more easily adapted than starting from scratch. Since it's a more advanced config section I suppose we should just leave it then. :-)